starter-code/index.html

```diff
@@ -8,6 +8,257 @@
+                <img class = "votes" src="../images/votes.png" />
+                <p>82</p>
+              </div>
+              <div class="commentbutton">
+                <img class = "comments" src="../images/comments.png" />
+                <p>8</p>
+              </div>
+              <p class = "save">SAVE</p>
+            </div>
+            <div class = "rightbuttons">
+              <div class ="category">
+                <p>ANDROID</p>
+              </div>
+              <p>+ 3</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class = "pop1">
+        <img class = "duo" src = "../images/today/app-2.png" />
+        <div class = "popcolumn">
+          <div>
+            <h2>Ask Gary Vee</h2>
+            <p class = "description">The unofficial Gary Vee persona chatbot on Messenger!</p>
+          </div>
+          <div class="buttons">
+            <div class = "leftbuttons">
+              <div class="votebutton">
+                <img class = "votes" src="../images/votes.png" />
+                <p>21</p>
+              </div>
+              <div class="commentbutton">
+                <img class = "comments" src="../images/comments.png" />
+                <p>9</p>
+              </div>
+              <p class = "save">SAVE</p>
+            </div>
+            <div class = "rightbuttons">
+              <div class ="category">
+                <p>GET SHIT DONE</p>
+              </div>
+              <p>+ 1</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class = "pop1">
+        <img class = "duo" src = "../images/today/app-3.png" />
+        <div class = "popcolumn">
+          <div>
+            <h2>The Electron Wheel</h2>
+            <p class = "description">Turn your standard bicyle into an electric bike.</p>
+          </div>
+          <div class="buttonslast">
+            <div class="votebutton">
+              <img class = "votes" src="../images/votes.png" />
+              <p>23</p>
+            </div>
+            <div class="commentbutton">
+              <img class = "comments" src="../images/comments.png" />
+              <p>2</p>
+            </div>
+              <p class = "save">SAVE</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+<!-- Right Column -->
+  <div class = "columncontainer">
+    <div class = "kittybot">
+      <h2>Kittybot for Slack.</h2>
+      <p>A friendly feline that sends you cool things.</p>
+      <p class = "kittybutton">ADD KITTYBOT TO SLACK</p>
+      <img class = "closekitty" src="../images/close.png" />
+    </div>
+    <div class ="livechat">
+      <p>UPCOMING LIVE CHATS</p>
+      <div class = "names">Adam Besvinick</div>
+      <div class = "names">Susan David</div>
+      <div class = "names">Avid Larizadeh Duggan</div>
+      <div class = "names">Katerina Markov Schneider</div>
+      <div class = "names">Jocelyn Glei</div>
+    </div>
+  </div>
+  </main>
 </body>
 </html>
```

Comment:
Good job with the overall html structure, use of classes, and . One thing I can note on though, I would like to see you use some the different structural tags that html5 gives us like: 

- <nav></nav>
- <section></section>
- <footer></footer>
- etc.

Using those tags will not only be more visually appealing because they give html more order and meaning than just having divs as containers for everything, but they will also come into play when you design actual pages for the internet as the use of those more descriptive tags help with your page ranking on search engines and it is also a best programming practice when it comes to developing accessible sites for visually impaired user who use screen readers.